### PR TITLE
Add a command to fetch the includes graph for a TU

### PR DESCRIPTION
**Pull request by lukka**

### CODE REVIEW

File: Extension/src/LanguageServer/client.ts [hunk 1]
```diff
@@ -536,6 +536,16 @@ interface DidChangeActiveEditorParams {
...
 }
 
+interface GetFileIncludeDataParams
+{
+    maxDepth: number;
+}
+
+interface IncludesGraphResult
+{
+    included_files: string[];
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/client.ts]: This should be includedFiles, right?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/src/LanguageServer/client.ts (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/src/LanguageServer/client.ts]: ...also I don't think the interfaces match the naming in cpptools.

Comment (lukka) [reply Extension/src/LanguageServer/client.ts]: Fixed, thank you

File: Extension/src/LanguageServer/client.ts [hunk 2]
```diff
@@ -536,6 +536,16 @@ interface DidChangeActiveEditorParams {
     selection?: Range;
 }
 
+interface GetFileIncludeDataParams
+{
+    maxDepth: number;
+}
+
+interface IncludesResult
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/client.ts]: GetIncludesResult?

File: Extension/src/LanguageServer/client.ts [hunk 3]
```diff
@@ -536,6 +536,16 @@ interface DidChangeActiveEditorParams {
     selection?: Range;
 }
 
+interface GetFileIncludeDataParams
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/client.ts]: GetIncludesParams?

Comment (lukka) [reply Extension/src/LanguageServer/client.ts]: Fixed, thank you

File: Extension/src/LanguageServer/client.ts (hunk 2 above)
Comment (lukka) [reply Extension/src/LanguageServer/client.ts]: fixed, thank you

**Review by sean-mcmanus — APPROVED** [review]

**Review by Colengms — APPROVED** [review]

**Pull request merged**